### PR TITLE
feat(smart-contracts): UnlockSwapPurchaser deployments

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This PR adds deployment script and info about `UnlockSwapPurchaser` deployments

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: packages/networks/src/networks/bsc.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export const bsc: NetworkConfig = {
     decimals: 18,
     coingecko: 'binancecoin',
   },
+  swapPurchaser: '0xeC83410DbC48C7797D2f2AFe624881674c65c856',
```
Comment (clemsos) [inline packages/networks/src/networks/bsc.ts:41]: actually uniswap isnt on bsc...

File: smart-contracts/scripts/all_networks [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
```
Comment (clemsos) [inline smart-contracts/scripts/all_networks:1]: need to rebase after merging https://github.com/unlock-protocol/unlock/pull/11045

Comment (julien51) [reply smart-contracts/scripts/all_networks:1]: Good, approved that other one so we can work on this here!

File: packages/networks/src/networks/gnosis.ts [hunk 3]
```diff
@@ -42,6 +42,7 @@ export const gnosis: NetworkConfig = {
     coinbase: 'DAI',
   },
   startBlock: 19338700,
+  swapPurchaser: '0x72381052e4F7765A00a403891420BF75876c75bB',
```
Comment (julien51) [inline packages/networks/src/networks/gnosis.ts:45]: Oh! is that v3?

**Review by julien51 — APPROVED** [review]
Very cool! Please check the Gnosis Safe one just to make sure as I am not sure it is v3?

File: smart-contracts/scripts/all_networks (hunk 2 above)
Comment (clemsos) [reply smart-contracts/scripts/all_networks]: ok done

File: packages/networks/src/networks/gnosis.ts (hunk 3 above)
Comment (clemsos) [reply packages/networks/src/networks/gnosis.ts:45]: no... actually uniswap is not deployed on gnosis either :/ Like bsc, it was triggered by the "deploy all" script

Comment (clemsos) [inline packages/networks/src/networks/gnosis.ts:45]:
```suggestion
```

File: packages/networks/src/networks/bsc.ts (hunk 1 above)
Comment (clemsos) [inline packages/networks/src/networks/bsc.ts:41]:
```suggestion
```

**Comment by clemsos** [comment]
> Very cool! Please check the Gnosis Safe one just to make sure as I am not sure it is v3?

yes, not v3 isnt deployed rn. I removed gnosis and bsc addresses from networks package for now. In case, we still have the address in this PR

**Pull request merged**